packages/app/components/drop/drop-free.tsx

```diff
@@ -71,8 +71,8 @@ export const DropFree = () => {
   const { rudder } = useRudder();
   const { data: userProfile } = useMyInfo();
 
-  const isVerified = false;
-  userProfile?.data?.profile.has_verified_phone_number &&
+  const isVerified =
+    userProfile?.data?.profile.has_verified_phone_number ||
     userProfile?.data?.profile.verified;
```

Comment:
Are you sure that an OR clause is correct here? Is a verified phone number really considered a verified user? I don't think so

Reply:
yeah, i have changed https://github.com/showtime-xyz/showtime-frontend/pull/1826#discussion_r1100670822